### PR TITLE
WW-570 read dropdown styling

### PR DESCRIPTION
**Pull request by cormorando**
https://wikia-inc.atlassian.net/browse/WW-570

//cc @Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js [hunk 1]
```diff
@@ -10,13 +10,27 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
...
+			$('*[data-tracking=' + $(this).data('category') + ']').parent('.rwe-page-header-nav__element')
+				.addClass('rwe-page-header-nav__element--active');
+			$(this).show();
+		}, function () {
+			$('*[data-tracking=' + $(this).data('category') + ']').parent('.rwe-page-header-nav__element')
+				.removeClass('rwe-page-header-nav__element--active');
+			$(this).hide();
+		});
+
+		$('.rwe-page-header-nav__link').on('click', function (e) {
```
Comment (WikiaTech) [inline extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js]: MAJOR Remove the unused function parameter "e". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: extensions/wikia/RWEPageHeader/styles/_dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+@import 'variables';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+
+.rwe-page-header-nav__dropdown {
+	display: none;
+	margin-top: -$content-spacing-medium;
+	position: absolute;
+	z-index: 99;
+	left: 50%;
```
Comment (sebastianmarzjan) [inline extensions/wikia/RWEPageHeader/styles/_dropdown.scss]: Nitpick (in an experiment), but for easier adjustments please follow SCSS guidelines. This relates to alphabetical sorting of properties as well as newline guidelines.

File: extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js [hunk 3]
```diff
@@ -10,13 +10,7 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
...
 
 	$(function () {
-		$('.rwe-page-header--dropdown').hover(function () {
-			$(this).addClass('rwe-page-header--active');
-		}, function () {
-			$(this).removeClass('rwe-page-header--active');
-		});
-
-		$('.rwe-page-header-nav__link').on('click', function () {
+		$('.rwe-page-header-nav__link').on('click', function (e) {
```
Comment (WikiaTech) [inline extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js:13]: MAJOR Remove the unused function parameter "e". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Review by lukaszkonieczny — APPROVED** [review]

File: extensions/wikia/RWEPageHeader/styles/_dropdown.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
+@import 'variables';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+
+.rwe-page-header-nav__dropdown {
+	display: none;
+	margin-top: 13px;
```
Comment (rybmat) [inline extensions/wikia/RWEPageHeader/styles/_dropdown.scss:7]: magic number

File: extensions/wikia/RWEPageHeader/styles/_dropdown.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+	.rwe-page-header-nav__dropdown-first-level {
+		@include display-inline-flex();
+		background-color: $background-color;
+		box-shadow: 0 $content-spacing-small $content-spacing-medium-minus 0 rgba(26, 26, 26, .3);
```
Comment (rybmat) [inline extensions/wikia/RWEPageHeader/styles/_dropdown.scss]: can we extract this color to _variables.scss? within the _search.scss the same value for box-shadow is used

**Pull request merged**